docs/development.md

```diff
@@ -29,17 +29,33 @@ open Package.swift
 
 
 Develop on Linux natively:
+
+It is easiest to use ubuntu 22.04 since this is the only Linux where swift 5.6+ is supported natively.
 ```bash
 git clone https://github.com/open-meteo/open-meteo.git
 cd open-meteo
 
 # Install the swift compiler as pointed out in the Vapor development guide
-apt install libnetcdf-dev libeccodes-dev libbz2-dev build-essential cdo python3-pip curl
+sudo apt install libnetcdf-dev libeccodes-dev libbz2-dev build-essential cdo python3-pip curl
+sudo apt-get install binutils git gnupg2 libc6-dev libcurl4-openssl-dev libedit2 libgcc-9-dev libpython3.8 \
+  libsqlite3-0 libstdc++-9-dev libxml2-dev libz3-dev pkg-config tzdata unzip zlib1g-dev
+sudo apt install libbz2-dev libz-dev
+
 pip3 install cdsapi
+wget https://download.swift.org/swift-5.8.1-release/ubuntu2204/swift-5.8.1-RELEASE/swift-5.8.1-RELEASE-ubuntu22.04.tar.gz
+tar xvzf swift-5.8.1-RELEASE-ubuntu22.04.tar.gz
+sudo mv swift-5.8.1-RELEASE-ubuntu22.04 /opt
+ln -s /opt/swift-5.8.1-RELEASE-ubuntu22.04/ /opt/swift
+echo 'export PATH=/opt/swift/usr/bin:$PATH' >> ~/.bashrc
+source ~/.bashrc
```

Comment:
Could you use the installation instructions from https://www.swiftlang.xyz? This should also install all the required dependencies like libsqlite automatically

Reply:
In fact, there is a pre-compiler switch that will be somehow affected:
https://github.com/the78mole/open-meteo/blob/6af4059058729820d55c2a222b809cd5725e3fa8/Sources/CTurboPFor/src/transpose.c#L315-329

But I'm looking forward for the results of your performance tests :-)